pkg/manager/impl/util/data.go

```diff
@@ -22,14 +22,17 @@ func shouldFetchOutputData(config *runtimeInterfaces.RemoteDataConfig, urlBlob a
 	return len(outputURI) > 0 && shouldFetchData(config, urlBlob)
 }
 
-// Returns an inputs URL blob and if config settings permit, inline inputs data for an execution.
+// GetInputs returns an inputs URL blob and if config settings permit, inline inputs data for an execution.
 func GetInputs(ctx context.Context, urlData dataInterfaces.RemoteURLInterface,
 	remoteDataConfig *runtimeInterfaces.RemoteDataConfig, storageClient *storage.DataStore, inputURI string) (
 	*core.LiteralMap, *admin.UrlBlob, error) {
+	var inputsURLBlob admin.UrlBlob
+	var fullInputs core.LiteralMap
+
 	if len(inputURI) == 0 {
-		return nil, nil, nil
+		return &fullInputs, &inputsURLBlob, nil
```

Comment:
aren't these values nil here?

Reply:
They are not nil.. they are default values of the struct types... which are valid struct objects with default values for their fields... I think that more accurately represent the intended behavior, if there is no inputURI, inputs are empty...